handbook/engineering/web/frontend-platform/wildcard-component-library.md

```diff
@@ -0,0 +1,66 @@
+6. When changes are approved, the product team merges the PR
+
+Important: we would like to avoid the situation where the FE Platform Team is engaged only at the end of the process or doesn’t know about the changes at all.
+
+### Process of changing or creating a component by the FE Platform Team or the Design Team
+It is possible that the Design Team or the FE Platform Team themselves initiate changes to the Standard UI Component Library. Here are the steps to take:
+
+1. The designer creates a Figma file with full specification regarding the component (separate from the general project file or general design system files)
+2. The designer initiates the cross-team communication by marking as reviewers:
+- The FE Platform Team engineers
+- At least one other Design Team member
+- Members of other teams affected by the changes 
+3. After all the reviews are completed, the FE Platform Team plans the implementation
+4. During the implementation, the FE Platform Team stays in constant communication with the Design Team members engaged in the project. Asks for review, if necessary.
+5. The FE Platform Team should ask for final reviews from:
+- The FE Platform Team members
+- The Design Team members engaged in the project
+6. Members of other teams engaged by the project and affected by the changes 
+7. When changes are approved, the FE Platform Team team merges the PR
+
+If product team after having started working on a feature, identifies in code review that a part of the UI should be extracted into a shared component (e.g., "This is very similar to the widget used in XYZ...can we make it a shared component?"), do not block the PR and kick off the "changing or creating a component" flow asynchronously.
+
+## Definitions
+### Full design specification should include:
+- All variants of the component (ex. Filled/empty or for auth/unauth user)
+- Clear descriptions of component behaviour added as notes on Figma
+- Clear descriptions of specific changes added as notes on Figma. This is primarily for changes to a design which might not be immediately obvious to a viewer, like a small change in the positioning of a button, a wording change etc
+- Designs or descriptions for different component states (e.g. hover/focus/animations). If it’s not possible to accurately represent an animation on Figma we should add a note to the design file explaining it (or linking to an example).
+- Light/Dark mode designs (specially important when changing colours)
+- Mobile/Desktop designs
+
+### A component is deemed to be ready from an engineering perspective when effort has been made to ensure it is:
+- Unit tested
+- Accessible
+- Documented textually, explaining when this component should be used and why
+- In Storybook
+- Built with a clear case for reusability
+
+## Sharing updates to the components system
+To keep other teams informed about the changes to the component system, we should share updates:
+- In the FE Platform Team weekly email
```

Comment:
This is no longer accurate

Reply:
How about:
- In the FE Platform team [status update](https://about.sourcegraph.com/handbook/engineering/leadership#status-updates)

Reply:
LGTM, updated my comment with a suggestion to apply.